centreon/src/Core/Notification/Application/UseCase/AddNotification/Factory/NotificationResourceFactory.php

```diff
@@ -63,15 +59,7 @@ public function __construct(
     public function create(NotificationResourceRepositoryInterface $repository, array $resource): NotificationResource
     {
         $resourceIds = array_unique($resource['ids']);
-
-        if ($this->user->isAdmin()) {
-            // Assert IDs validity without ACLs
-            $existingResources = $repository->exist($resourceIds);
-        } else {
-            // Assert IDs validity with ACLs
-            $accessGroups = $this->readAccessGroupRepository->findByContact($this->user);
-            $existingResources = $repository->existByAccessGroups($resourceIds, $accessGroups);
-        }
```

Comment:
I still think this is a really bad idea to remove any acl filters, here any users could have access to any resources (BV or Not)

Reply:
> I still think this is a really bad idea to remove any acl filters, here any users could have access to any resources (BV or Not)

Do you have any suggestion ?

Reply:
this should not be removed, then used a dedicated repository to get BV

Reply:
> this should not be removed, then used a dedicated repository to get BV


beside BV the probleme is also on host group and service group not only the BV